akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -656,14 +672,17 @@ private[projection] class R2dbcOffsetStore(
    * Completed with `true` if accepted, otherwise `false` if rejected or failed with `EnvelopeRejected` if
    * backtracking envelope is rejected.
    */
-  def isAccepted[Envelope](envelope: Envelope): Future[Boolean] = {
+  def isAccepted[Envelope](envelope: Envelope): Future[IsAcceptedResult] = {
```

Comment:
Following previous comment...

```scala
  def validate[Envelope](envelope: Envelope): Future[Validation]
```

Also, scaladoc needs to be adapted. Not a boolean anymore.